pom.xml

```diff
@@ -118,6 +118,18 @@
             <optional>true</optional>
         </dependency>
 
+        <dependency>
```

Comment:
are those really needed runtime dependencies? or do you use them for tests only? in which case they need to have test scope

Reply:
I removed them.
I'll make a separate branch for the validation tests that need them.  And will scope them to test.